plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1955,7 +1955,7 @@ bool producer_plugin_impl::process_unapplied_trxs( const fc::time_point& deadlin
 
             auto prev_billed_cpu_time_us = trx->billed_cpu_time_us;
             if( prev_billed_cpu_time_us > 0 && !_subjective_billing.is_account_disabled( first_auth ) && !rl.is_unlimited_cpu( first_auth )) {
-               uint64_t prev_billed_plus100_us = prev_billed_cpu_time_us + EOS_PERCENT( prev_billed_cpu_time_us, 100 * config::percent_1 );
+               int64_t prev_billed_plus100_us = prev_billed_cpu_time_us + EOS_PERCENT( prev_billed_cpu_time_us, 100 * config::percent_1 );
```

Comment:
This is narrowing from `uint64_t` to `int64_t` which should be fine here as `prev_billed_cpu_time_us` is `uint32_t`. BTW, this is a very odd way to multiple a value by 2. It is written this way because originally we had a different value than `100`.

Reply:
Thanks for the review. Yes, I was worried about that narrowing too. Since `fc::microseconds`'s `_count` is defined as `int64_t`, it should fine.